### ISSUE TITLE
Deprecation Warnings

### ISSUE THREAD

**Issue by Alexander-Shukaev** [matplotlib/matplotlib#12728]
```
import matplotlib
```

causes

```
matplotlib/__init__.py:107: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import MutableMapping
matplotlib/rcsetup.py:20: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Iterable, Mapping
matplotlib/rcsetup.py:20: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Iterable, Mapping
matplotlib/colors.py:53: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Sized
```

**Comment by timhoffm** [comment]
Already fixed in #12686.

**Comment by Alexander-Shukaev** [comment]
I see, sorry for the noise.
